jiant/scripts/download_data/dl_datasets/files_tasks.py

```diff
@@ -1012,3 +1036,70 @@ def download_ropes_data_and_write_config(
         },
         path=task_config_path,
     )
+
+
+def download_acceptability_judgments_data_and_write_config(
+    task_name: str, task_data_path: str, task_config_path: str
+):
+    name_map = {
+        "acceptability_definiteness": "definiteness",
+        "acceptability_coord": "coordinating-conjunctions",
+        "acceptability_whwords": "whwords",
+        "acceptability_eos": "eos",
+    }
+    dataset_name = name_map[task_name]
+    os.makedirs(task_data_path, exist_ok=True)
+    # data contains all train/val/test examples
+    # metadata contains the split indicators
+    # (there are 10 CV folds, we use fold1 by default, see below)
+    data_path = os.path.join(task_data_path, "data.json")
+    metadata_path = os.path.join(task_data_path, "metadata.json")
+    download_utils.download_file(
+        url="https://raw.githubusercontent.com/decompositional-semantics-initiative/DNC/master/"
```

Comment:
candidate to be a constant instead of duplicated string

Reply:
This one is odd because it's part of a string, broken into two because of length, and the second half does differ between the two. Given that it's two explicit URLs, I would lean towards keeping them as is.